projects/contrax-finance/index.js

```diff
@@ -59,6 +54,7 @@ async function getSteerData(api) {
     api.multiCall({
       abi: "uint256:balance",
       calls: STEER_VAULTS,
+      permitFailure: true
```

Comment:
shouldn't we add a check if empty on line 62? else it can have NaN values?

Reply:
yea certainly, wdyt about something like :

```
  for (const [i, bal] of bals.entries()) {
    if (!bal || !supplies[i] || !reserves[i] || !reserves[i][0] || !reserves[i][1]) continue
    const ratio = bal / supplies[i];
    const token0Bal = reserves[i][0] * ratio;
    const token1Bal = reserves[i][1] * ratio;
    api.addToken(token0s[i], token0Bal);
    api.addToken(token1s[i], token1Bal);
  }
```

to prevent mapping error if multicall is returning null

Reply:
yes, that should be enough since there are no permitFailure for other multicalls